pandas/io/formats/format.py

```diff
@@ -95,6 +95,7 @@
         CompressionOptions,
         FilePath,
         FloatFormatType,
+        IntegerFormatType,
```

Comment:
I'm surprised this worked because `_typing.py` has no `IntegerFormatType`

Reply:
Oh did not notice that until I created a PR, just pushed something that fixes that.